### PR TITLE
Add list view

### PR DESCRIPTION
**Pull request by TotalCaesar659**
Here is list view draft, fixes https://github.com/sharkwouter/minigalaxy/issues/329.

### CODE REVIEW

**Comment by TotalCaesar659** [comment]
@sharkwouter Could you check this?

**Comment by sharkwouter** [comment]
The code is pretty good. I tried it and rebased the code on master. I do feel it's a bit of a waste that there isn't more information on the screen, though. It makes this view seem like a bit wasteful with space. I also found that at 1440p, it will have 2 rows next to each other. I'm not sure if that's intended or not.

**Comment by TotalCaesar659** [comment]
> I also found that at 1440p, it will have 2 rows next to each other. I'm not sure if that's intended or not.

It was not intended, I didn't see it on my old 4:3 display, but yeah, there are new columns appear when expanding the window manually. Strange, I can't find where it is.

> I do feel it's a bit of a waste that there isn't more information on the screen, though. It makes this view seem like a bit wasteful with space.

This empty space can be filled later by some additional info: description, played time etc. when it'll be implemented. I didn't want to complicate it more, this is a thing for another big PR (customizable table or smth for varying displays).

File: minigalaxy/ui/gametile.py [hunk 1]
```diff
@@ -566,3 +566,561 @@ def update_to_state(self, state):
         self.current_state = state
         if state in self.STATE_UPDATE_HANDLERS:
             self.STATE_UPDATE_HANDLERS[state](self)
+
+
+@Gtk.Template.from_file(os.path.join(UI_DIR, "gametilelist.ui"))
+class GameTileList(Gtk.Box):
```
Comment (sharkwouter) [inline minigalaxy/ui/gametile.py:572]: Could this be put in its own file or would that cause issues?

Comment (TotalCaesar659) [reply minigalaxy/ui/gametile.py]: Loos like it works, don't see any bugs.

**Comment by sharkwouter** [comment]
Thanks for the work. I'll merge this after version 1.1.0 has been released.

**Comment by TotalCaesar659** [comment]
Great, is 1.1.0 planned to be released soon? it's been a long time. Maybe it's better to delay [enhancements](https://github.com/sharkwouter/minigalaxy/milestone/6) to 1.2.0 and focus on bugs.

**Comment by sharkwouter** [comment]
I think now would be the time to merge this, but there was still a test which didn't pass, so the other PR went in before this. Would it be possible to rebase and fix the issue that caused the test to fail?

**Comment by TotalCaesar659** [comment]
@sharkwouter Could you check tests? I don't know how to fix them.

**Comment by sharkwouter** [comment]
Thanks!

**Pull request merged**